newdle/api.py

```diff
@@ -181,14 +190,22 @@ def get_my_newdles():
 @api.route('/newdle/', methods=('POST',))
 @use_kwargs(NewNewdleSchema(), locations=('json',))
 def create_newdle(title, duration, timezone, timeslots, participants):
+    participants = [(p.pop('signature'), Participant(**p)) for p in participants]
+
+    for signature, p in participants:
```

Comment:
no idea if it looks better, but `if not all(check_user_signature(...) for sig, p in participants)`

Reply:
It ends up being harder to read, TBH...